doc/apiv3/endpoints/users.apib

```diff
@@ -169,20 +169,6 @@ When calling this endpoint the client provides a single object, containing at le
                 "message": "You are not allowed to update the account of this user."
             }
 
-+ Response 404 (application/hal+json)
```

Comment:
What happens then if I try to patch `/api/v3/users/42` when user `42` doesn't exist?